addons/kotsadm/alpha/tmpl-kotsadm-api-proxy.yaml

```diff
@@ -11,4 +11,4 @@ spec:
         - name: HTTP_PROXY
           value: ${PROXY_ADDRESS}
         - name: NO_PROXY
-          value: ${NO_PROXY_ADDRESSES},${OBJECT_STORE_CLUSTER_IP}
+          value: ${NO_PROXY_ADDRESSES},${OBJECT_STORE_CLUSTER_IP},${KUBERNETES_CLUSTER_IP}
```

Comment:
There's graphql call that creates a Secret with this IP and ignores the service cidr